src/lib/operators/validate.cpp

```diff
@@ -165,18 +199,26 @@ void Validate::_validate_chunks(const std::shared_ptr<const Table>& in_table, co
         output_segments.push_back(ref_segment_out);
       }
 
-      // Otherwise we have a Value- or DictionarySegment and simply iterate over all rows to build a poslist.
+      // Otherwise we have a non-reference Segment and simply iterate over all rows to build a poslist.
```

Comment:
The "otherwise" is now too far away from the initial comment to still make sense to a reader.

Reply:
I thought the following `else` adds enough context. What do you suggest, simply removing the "otherwise"?

Reply:
never mind